en/docs/consuming-services/consuming-a-sevice.md

```diff
@@ -47,106 +44,101 @@ The overview page of an API displays the subscribed versions of the API along wi
-
--  **Allowed grant types**:  Code
--  **Public client**: Mark as a public client in an application.
--  **Authorized redirect URLs**: Enter the web app URL you copied earlier.
--  **Allowed origins** field: Add the same URLs that you added as authorized redirect URLs.
--  **Access Token**: Set  JWT as the Token type.
+To invoke the API/service, you need an access token. Create an OAuth application in the IdP (e.g., Asgardeo) with the following settings:
 
-Choreo uses Asgardeo as the default identity provider for Choreo applications. When you create an application in the Choreo Developer Portal, it automatically creates a corresponding application in Asgardeo. You can go to the Asgardeo application to specify the configurations required for end users to sign in to the front-end application. Follow the steps below to configure the Asgardeo OAuth application:
+- **Allowed grant types**: Code
+- **Public client**: Enable this option.
+- **Authorized redirect URLs**: Add the web app URL.
+- **Allowed origins**: Add the same URLs as authorized redirect URLs.
+- **Access Token**: Set to JWT.
 
-1. Access Asgardeo at [https://console.asgardeo.io/](https://console.asgardeo.io/) and sign in with the same credentials with which you signed in to Choreo.
-2. Make sure you are in the same organization that you were when you created the application in the Choreo Developer Portal. You can click the **Organization** list in the Asgardeo Console top menu and ensure you are in the correct organization.
-3. In the Asgardeo Console's left navigation, click **Applications**. You will see the **readingListApp** that Choreo automatically created for you.
-4. Click on the edit icon to edit the application.
-5. Click the **Protocol** tab and apply the following changes:
+Choreo uses Asgardeo as the default IdP. When you create an application in the Choreo Developer Portal, it automatically creates a corresponding application in Asgardeo. Follow these steps to configure the Asgardeo OAuth application:
 
+1. Sign in to [Asgardeo](https://console.asgardeo.io/) using the same credentials as Choreo.
+2. Ensure you’re in the same organization used in the Choreo Developer Portal.
+3. In the Asgardeo Console, click **Applications** in the left navigation. You’ll see the **readingListApp** created by Choreo.
+4. Click the edit icon to edit the application.
+5. Go to the **Protocol** tab and make the following changes:
     1. Under **Allowed grant types**, select **Code**.
-    2. Select the **Public client** checkbox.
-    3. In the **Authorized redirect URLs** field, enter the web app URL you copied earlier and click the **+** icon to add the entry.
-    4. In the **Allowed origins** field, add the same URLs that you added as authorized redirect URLs.
-    5. Under **Access Token**, select **JWT** as the **Token type**.
+    2. Enable the **Public client** checkbox.
```

Comment:
For checkboxes we use the terms select and clear. https://developers.google.com/style/ui-elements#term-checkbox